src/libraries/System.Net.Ping/src/System/Net/NetworkInformation/IcmpV4MessageConstants.cs

```diff
@@ -77,7 +77,7 @@ public static IPStatus MapV4TypeToIPStatus(int type, int code)
                     return IPStatus.SourceQuench;
 
                 case IcmpV4MessageType.TimeExceeded:
-                    return IPStatus.TimeExceeded;
+                    return IPStatus.TtlExpired;
```

Comment:
briefly looking at ICMPv4, it also the code 1 => Fragment reassembly time exceeded. I'm not sure if that matters but we may make is same as IPv6 TtlReassemblyTimeExceeded.

Reply:
IcmpV4MessageType has no enum variant called `FragmentReassemblyTimeExceeded`